diff --color -ruN '--exclude=*.rej' '--exclude=*.orig' a/CMakeLists.txt b/CMakeLists.txt
--- a/CMakeLists.txt	2026-09-23 12:45:14.805257310 +0500
+++ b/CMakeLists.txt	2026-09-23 12:45:14.855413249 +0500
@@ -364,10 +364,6 @@
 
 # temporary workaround for a gfortran v7.3 ICE on Fedora 27 64-bit
 
-set (sqlite3_CSRCS
-  vendor/sqlite3/sqlite3.c
-  )
-
 set (wsjt_CSRCS
   lib/gran.c
   lib/init_random_seed.c
@@ -403,7 +399,6 @@
   )
 
 set (all_C_and_CXXSRCS
-  ${sqlite3_CSRCS}
   ${wsjt_CSRCS}
   ${all_CXXSRCS}
   )
@@ -577,6 +572,8 @@
   find_package (Qt5AxContainer REQUIRED)
 endif (WIN32)
 
+pkg_check_modules(SQLITE3 sqlite3>=3.26 REQUIRED)
+message (STATUS "SQLITE3_LIBRARIES: ${SQLITE3_LIBRARIES}")
 
 #
 # Library building setup
@@ -896,9 +893,9 @@
     #   LINK_FLAGS -Wl,--stack,16777216
     #   )
   endif ()
-  target_link_libraries (js8 wsjt_fort_omp wsjt_cxx Qt5::Core)
+  target_link_libraries (js8 wsjt_fort_omp wsjt_cxx Qt5::Core ${SQLITE3_LIBRARIES})
 else (${OPENMP_FOUND} OR APPLE)
-  target_link_libraries (js8 wsjt_fort wsjt_cxx Qt5::Core)
+  target_link_libraries (js8 wsjt_fort wsjt_cxx Qt5::Core ${SQLITE3_LIBRARIES})
 endif (${OPENMP_FOUND} OR APPLE)
 
 # if (UNIX)
diff --color -ruN '--exclude=*.rej' '--exclude=*.orig' a/Inbox.h b/Inbox.h
--- a/Inbox.h	2026-09-23 12:45:14.806893960 +0500
+++ b/Inbox.h	2026-09-23 12:45:14.855365153 +0500
@@ -10,7 +10,7 @@
 #include <QPair>
 #include <QVariant>
 
-#include "vendor/sqlite3/sqlite3.h"
+#include "sqlite3.h"
 
 #include "Message.h"
 
